### ISSUE TITLE
Please use guthub releases and tags

### ISSUE THREAD

**Issue by luzpaz** [jwdj/EasyABC#74]
It makes it easier to track changes for packagers.

**Comment by revad** [comment]
My understanding is that EasyABC is developed here and released on sourceforge .
https://sourceforge.net/projects/easyabc/files/EasyABC/
If that's true - or if it is not - a clarification of the relationship between the two sites would be useful and declaring a release here corresponding to an SF release would be helpful.

From reading discussions on https://groups.io/g/abcusers/ and https://thesession.org/discussions/ it's clear that SF is where ordinary users - musicians - expect to get it from and that they don't understand github - which is unsurprising. But it seems many don't trust SF because it apparantly gives them malware alerts. See this recent thread for example: https://thesession.org/discussions/48519

**Comment by luzpaz** [comment]
So many FLOSS music applications migrated to github from SF, it is surprising the community has stayed that long.

**Comment by aupfred** [comment]
Yes it is confirmed that when a release is formalized, @jwdj and Seymour put in Sourceforge the release.
Might help in any case to also tag the release here
